specification/eventgrid/data-plane/Microsoft.ApiManagement/stable/2018-01-01/APIManagement.json

```diff
@@ -0,0 +1,161 @@
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/<ResourceType>/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "ProductCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.ProductCreated event.",
+      "type": "object",
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/<ResourceType>/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "ProductUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.ProductUpdated event.",
+      "type": "object",
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/<ResourceType>/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "ProductDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.ProductDeleted event.",
+      "type": "object",
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/<ResourceType>/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "APICreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.APICreated event.",
+      "type": "object",
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/<ResourceType>/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "APIUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.APIUpdated event.",
```

Comment:
I noticed that for some events `API` is capitalized, and in other cases, it's spell `Api` (e.g. `APIUpdated` vs `ApiReleaseUpdated`)  I just want to confirm that the difference in casing between these event is expected.

Reply:
Also, curious about this - We would likely need to rename this via swagger transforms in .NET. It also seems odd that we have API (in whatever casing) twice in the name of the events.

Reply:
The service is ApiManagement, and the event is that an API has been updated, so it actualle seems normal to me?

Reply:
Yeah, that makes sense but the casing should be consistent, right?